cmd/genji/shell/command.go

```diff
@@ -2,20 +2,13 @@ package shell
 
 import (
 	"fmt"
-	"sort"
-
+	"github.com/agnivade/levenshtein"
 	"github.com/genjidb/genji"
 	"github.com/genjidb/genji/document"
 )
 
-var commands = map[string]string{
```

Comment:
Before, we had one place where we kept the commands, now we have 3 places we need to update to add them. Why not keeping this map and extract + sort the keys whenever we need them?